polygeist/tools/cgeist/Test/Verification/sycl/types.cpp

```diff
@@ -2,56 +2,53 @@
 
 #include <sycl/sycl.hpp>
 
-// CHECK-DAG: !sycl_id_1_ = !sycl.id<1>
-// CHECK-DAG: !sycl_id_2_ = !sycl.id<2>
-// CHECK-DAG: !sycl_range_1_ = !sycl.range<1>
-// CHECK-DAG: !sycl_range_2_ = !sycl.range<2>
-// CHECK-DAG: !sycl_accessor_1_i32_read_write_global_buffer = !sycl.accessor<[1, i32, read_write, global_buffer], (!sycl.accessor_impl_device<[1], (!sycl.id<1>, !sycl.range<1>, !sycl.range<1>)>, !llvm.struct<(ptr<i32, 1>)>)>
-// CHECK-DAG: !sycl_accessor_2_i32_read_write_global_buffer = !sycl.accessor<[2, i32, read_write, global_buffer], (!sycl.accessor_impl_device<[2], (!sycl.id<2>, !sycl.range<2>, !sycl.range<2>)>, !llvm.struct<(ptr<i32, 1>)>)>
+// CHECK-DAG: ![[ACCESSOR:.*]] = !sycl.accessor<[1, i32, read_write, global_buffer], (!sycl.accessor_impl_device<[1], (!sycl.id<1>, !sycl.range<1>, !sycl.range<1>)>, !llvm.struct<(ptr<i32, 1>)>)>
+// CHECK-DAG: ![[ACCESSOR1:.*]] = !sycl.accessor<[2, i32, read_write, global_buffer], (!sycl.accessor_impl_device<[2], (!sycl.id<2>, !sycl.range<2>, !sycl.range<2>)>, !llvm.struct<(ptr<i32, 1>)>)>
+// CHECK-DAG: ![[ACCESSOR2:.*]] = !sycl.accessor<[3, f32, read_write, global_buffer], (!sycl.accessor_impl_device<[3], (!sycl.id<3>, !sycl.range<3>, !sycl.range<3>)>, !llvm.struct<(ptr<f32, 1>)>)>
```

Comment:
To match its dimensions: 
```suggestion
// CHECK-DAG: ![[ACCESSOR1:.*]] = !sycl.accessor<[1, i32, read_write, global_buffer], (!sycl.accessor_impl_device<[1], (!sycl.id<1>, !sycl.range<1>, !sycl.range<1>)>, !llvm.struct<(ptr<i32, 1>)>)>
// CHECK-DAG: ![[ACCESSOR2:.*]] = !sycl.accessor<[2, i32, read_write, global_buffer], (!sycl.accessor_impl_device<[2], (!sycl.id<2>, !sycl.range<2>, !sycl.range<2>)>, !llvm.struct<(ptr<i32, 1>)>)>
// CHECK-DAG: ![[ACCESSOR3:.*]] = !sycl.accessor<[3, f32, read_write, global_buffer], (!sycl.accessor_impl_device<[3], (!sycl.id<3>, !sycl.range<3>, !sycl.range<3>)>, !llvm.struct<(ptr<f32, 1>)>)>
```